muesli/web/templates/Fragments/menu.pt

```diff
@@ -34,6 +34,15 @@
                  class="nav-link" tal:attributes="href request.route_path('start')">Übersicht</a>
             </li>
 
+            <li class="nav-item active">
+              <a class="nav-link" tal:attributes="href request.route_path('lecture_list')">Vorlesungen</a>
+            </li>
+
+            <li class="nav-item active">
+              <a tal:condition="request.userInfo.is_loggedin()"
+                 class="nav-link" tal:attributes="href request.route_path('user_api_keys')">API</a>
```

Comment:
I think this fills the navbar a bit. Could you move this back to the "Nutzereinstellungen" dropdown?
When editing lectures the space from the left most side the lectures entry in the navbar to the right edge of the screen is relevant because nested dropdowns are pretty wide. I think API does not need to be there.